docs-app/package.json

```
@@ -39,7 +39,7 @@
     "@glint/environment-ember-template-imports": "^1.0.2",
     "@glint/template": "^1.0.2",
     "@html-next/vertical-collection": "^4.0.2",
-    "@nullvoxpopuli/eslint-configs": "~2.2.59",
+    "@nullvoxpopuli/eslint-configs": "3.1.3",
```

```suggestion
    "@nullvoxpopuli/eslint-configs": "^3.1.3",
```

Any reason _not_ to use `^` for this instead?  All other dependencies here do so.

https://github.com/CrowdStrike/ember-toucan-core/blob/99c434424d1bef8a648f4548ed609981351864e2/packages/ember-toucan-core/package.json#L63